content/en/docs/reference/scheduling/config.md

```diff
@@ -26,7 +26,7 @@ struct.
 A minimal configuration looks as follows:
 
 ```yaml
-apiVersion: kubescheduler.config.k8s.io/v1beta1
+apiVersion: kubescheduler.config.k8s.io/v1beta2
```

Comment:
Replace v1beta1 with v1beta2 in L23?

BTW: @sftim as it's now v1beta2, with slightly different API fields, should we also update L7 to v1.22?

Reply:
Better listed the options.

I don't think we should update the min beta version.